test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/ObjectResultExecutorTest.cs

```diff
@@ -20,37 +20,6 @@ namespace Microsoft.AspNetCore.Mvc.Internal
 {
     public class ObjectResultExecutorTest
     {
-        [Fact]
```

Comment:
No problem with `ConnegScenario` in general but would prefer to see it introduced w/o `src` changes in the same PR. (Well, "conneg" is a fairly ugly shorthand but whatever.)